sickbeard/helpers.py

```diff
@@ -1684,6 +1687,9 @@ def download_file(url, filename, session=None, headers={}):
             except Exception:
                 logger.log(u"Problem setting permissions or writing file to: %s" % filename, logger.WARNING)
 
+    except ConnectionError as e:
+        logger.log(u"No internet available while loading download URL %s Error: %r" % (url, ex(e)))
```

Comment:
we already have this some row below so why we still have ConnectionError?

`except requests.exceptions.ConnectionError as e:`